Helper/Data.php

```diff
@@ -93,53 +92,53 @@ public function __construct(
     /**
      * Returns if category sorting is enabled
      *
-     * @param StoreInterface|null $store the store model or null.
+     * @param Store|null $store the store model or null.
      * @return bool the configuration value
      */
-    public function isCategorySortingEnabled(StoreInterface $store = null)
+    public function isCategorySortingEnabled(Store $store = null)
```

Comment:
Why change from the interface to the concrete type?

Reply:
Scope helper returns concrete type

Reply:
Changing the return type in the Helper will also require changes to the main module. Should I open parallel PR to this one?